### DISCUSSION TITLE
请教下： 如何实现 编写python 代码 并 执行调用 的 微调？

### DISCUSSION THREAD

**Discussion by tanguofu** [zai-org/ChatGLM3#490] [Q&A] [answered]
具体来说 当前已经有[有调用工具的多轮会话的微调](https://github.com/THUDM/ChatGLM3/tree/main/finetune_chatmodel_demo)， 但是 文中说没有实现基于代码的， 所以这里请教下， 有没有代码微调的案例呢？ 

非常感谢

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
没有代码案例，技术手册已经提到了不支持代码微调
